doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+Concept                       EEGLAB          			        MNE
+Importing data		`EEG = pop_fileio('/your/data.dat');`		raw = mne.io.read_raw('/your/data-raw.fif')
+
+Filtering raw data	`EEG = pop_eegfiltnew(EEG, 1, 40);`			raw.filter(1, 40)
+
+Run ICA				`EEG = pop_runica(EEG);`					ica = mne.preprocessing.ica.ica().fit(raw)
+
+
+Epoching data		`EEG_epochs = pop_epoch(EEG,				`epochs = mne.Epochs(raw,
+																events=mne.find_events(raw),
+                              {'1', '2'},		                event_id=dict(cond1=1, cond2=2),
+                              [-.2, .8]);`						tmin=-2, tmax=.8)`
+
+Selecting epochs    `EEG_epochs2 = pop_epoch(EEG_epochs,		`epochs_2 = epochs["cond2"]`
+                            {'2'});`
+
+
+
+ERPs															`evoked = epochs.average()`
+
+Plot Butterfly ERP	`pop_timtopo(EEG_epochs, ...);`             `evoked.jointplot()`
+
+Contrast ERPs		`pop_compareerps(EEG_epochs, EEG_epochs2);`	`(evoked - evoked2).plot()`
+
+Saving data         `EEG = pop_saveset(EEG,'filename',set);`	`[raw/epochs/evoked].save('/your/data-[raw/epo/evo].fif')
+
+============================================================================================================================
+
+
+Tips on specific features for people coming from EEGLAB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* The MNE event system stores onset, duration, and type (as integers). Beware of the find_events function; with its default settings, it will not find many events in typical EEG formats. Try using `shortest_event=0, min_duration=0, consecutive=True` if that happens to you.
+* The MNE resampling function is much slower than the EEGLAB equivalent. As a temporary work-around, many MNE functions allow a `decim` keyword to downsize analyses or objects. Also, remember that MNE can be more readily parallelized than MNE; via e.g. joblib, a whole experiment can be downsampled simultaneously.
+* The MNE `eog` channel type is generally conceptualized around bipolar eye channels. Channels marked as `eog` are not included in many analyses. Typical EEGLAB behavior is roughly equivalent to setting all EEG channels, including monopolar eye sensors, to channel type `eeg`.
+* MNE evoked objects support arithmetic; you can do `(epochs["critical"].average() - epochs["control"].average()).plot()`. However note that this is weighted by trial numbers! (Use `combine_evoked` to use uniform weights.) Probably the best way to plot multi-channel ERPs in MNE is `evoked.plot(spatial_colors=True)`.
+* To recreate the ERPimages sorted by response time typical of EEGLAB, reaction times can be found with the ... command, and stored in the second column of an `events` matrix. The plot can then be created with with `mne.viz.plot_epochs_image(epochs, ... order=events[:,1].argsort())`.
+* Beware that `mne.viz.plot_epochs_image` will by default plot all channels; use the `picks` kwarg, unless you want 128 figures popping up on your screen.
```

Comment:
The EEGLAB functions for stuff like this all require you to pre-specify the single channel (or component) you want to plot.